### PR TITLE
chore: add the mirror.yml

### PR DESCRIPTION
**Pull request by xingwanying**
### 👀 PR includes

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [x] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🔍 Self Check before Merge

- [ ] Add or update relevant Docs.
- [ ] Add or update relevant Demos.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

**Review by zxc0328 — APPROVED** [review]
LG

File: .github/workflows/mirror.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          # 注意替换为你的 GitHub 源仓库地址
+          source-repo: git@github.com:antvis/S2.git
+          # 注意替换为你的 Gitee 目标仓库地址
+          destination-repo: git@gitee.com:antv-s2/antv-s2.git
+
+      - name: ✅ Build Gitee Pages
+        uses: yanglbme/gitee-pages-action@master
+        with:
+          # 注意替换为你的 Gitee 用户名
+          gitee-username: afc163
```
Comment (lijinke666) [inline .github/workflows/mirror.yml:24]: 用户名用偏右的?

File: .github/workflows/mirror.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          # 注意在 Settings->Secrets 配置 GITEE_RSA_PRIVATE_KEY
+          SSH_PRIVATE_KEY: ${{ secrets.GITEE_RSA_PRIVATE_KEY }}
+        with:
+          # 注意替换为你的 GitHub 源仓库地址
+          source-repo: git@github.com:antvis/S2.git
+          # 注意替换为你的 Gitee 目标仓库地址
+          destination-repo: git@gitee.com:antv-s2/antv-s2.git
+
+      - name: ✅ Build Gitee Pages
+        uses: yanglbme/gitee-pages-action@master
```
Comment (qubaomingg) [inline .github/workflows/mirror.yml:21]: yanglbme 是谁啊

Comment (lijinke666) [reply .github/workflows/mirror.yml:21]: 开发这个action的作者啊

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**